docs/tutorial/batch-minting.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Batch-minting NFTs on Ethereum 🚧
+description: This tutorial addresses some techniques in batch-minting big numbers of Ethereum non-fungible tokens (NFTs) as well as common idioms of uploading files and metadata on to IPFS/Filecoin via NFT.storage.
+issueUrl: https://github.com/protocol/nft-website/issues/253
+related: 
+  'Lazy minting': https://nftschool.dev/tutorial/lazy-minting/
+---
+
+ # Batch-minting NFTs on Ethereum
+
+ This tutorial addresses some techniques in batch-minting big numbers of Ethereum non-fungible tokens (NFTs) as well as common idioms of uploading files and metadata on to IPFS/Filecoin via NFT.storage.
+
+See also: [Lazy minting](./lazy-minting.md)
+
+Quite often,  NFT projects consist of hundreds if not thousands of files and metadata starting their lives out on a computer file system, waiting nervously to be part of a mint.
+
+There are cases in which it is desirable to mint many NFTs all at once instead of doing each of them in a [just-in-time](./lazy-minting.md) fashion. Large single drops are often for more than just art, including scenarios like large PFP (profile picture) drops, event tickets, game packs, and etc.
+
+Some challenges that lie in minting a bulk of NFTs include high gas fees on the Ethereum network, synchronizing between mint and upload failures and recovering from them, and performance of uploading files to NFT.storage, etc.
+
+## Batch-uploading NFT assets
+
+Here is an overview of strategies to upload to NFT.storage in bulk.
```

Comment:
```suggestion
Here is an overview of strategies you can choose from to upload to NFT.storage in bulk. 

### Drag-and-drop uploading with NFTUp
[NFTUp](https://nft.storage/blog/post/2022-04-05-announcing-nftup/) is a downloadable application for adding data to NFT.Storage. Content creators can drag-and-drop metadata and assets, from individual files to large directories. Once uploaded, they are ready to be minted into NFTs by smart contracts and then traded on marketplaces or browsed in galleries.

```

add section for NFTUp — according to @dchoi27 it's been popular and works well for bulk uploads